lib/ansible/modules/windows/win_robocopy.ps1

```diff
@@ -43,17 +42,16 @@ Function SearchForError($cmd_output, $default_msg) {
     $separator_count = 0
     $error_msg = $default_msg
     ForEach ($line in $cmd_output) {
-        if (-Not $line) {
+        if (-not $line) {
             continue
         }
 
         if ($separator_count -ne 3) {
             if (Select-String -InputObject $line -pattern "^(\s+)?(\-+)(\s+)?$") {
                 $separator_count += 1
             }
-        }
-        Else {
-            If (Select-String -InputObject $line -pattern "error") {
+        } else {
+            if (Select-String -InputObject $line -pattern "error") {
```

Comment:
Good question. The logic was not modified from before, so there's no regression from the change.